core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -424,7 +425,7 @@ object ConfigCommand extends Config {
         case ConfigType.Topic =>
           adminClient.listTopics(new ListTopicsOptions().listInternal(true)).names().get().asScala.toSeq
         case ConfigType.Broker | BrokerLoggerConfigType =>
-          adminClient.describeCluster(new DescribeClusterOptions()).nodes().get().asScala.map(_.idString).toSeq :+ ConfigEntityName.Default
+          adminClient.describeCluster(new DescribeClusterOptions()).nodes().get().asScala.map(_.idString).toSeq :+ BrokerDefaultEntityName
```

Comment:
Please substitute `""` for `BrokerDefaultEntityName` in the `match` below, as well as in `getResourceConfig`.

Reply:
Thanks @bdbyrne . Good suggestion. I've updated it in this commit https://github.com/apache/kafka/pull/8675/commits/83cc2328099ef1424c3cea492156bd4a82f3fe65. Thank you.